news/2024/2024-05-10-kyoku-original-song-and-beatmapping-contest.md

```diff
@@ -0,0 +1,60 @@
+---
+
+All mappers and music producers, gather up! Creativity to the maximum using music is now possible within osu!
+
+![](/wiki/shared/news/2024-05-10-kyoku-original-song-and-beatmapping-contest/banner.jpg)
+
+Better yet, **you** can join in on the fun! A team of hosts, [bokeru](https://osu.ppy.sh/users/11308433), [chidodou](https://osu.ppy.sh/users/13868581), [Supportive](https://osu.ppy.sh/users/21482068) and [Whithard](https://osu.ppy.sh/users/24552396) have joined up to create a contest revolving around the idea of expression, versatility, and best of, **expression**! Keep reading to see how you can join and be an original participant of this contest!
+
+With the first iteration and the first of its kind in osu!, as well as being most inspired from BMS of Fighters (surely something you’ve heard about before), Kyoku is a contest that will focus on not only beatmapping, but song production as well. Instead of free song choice, or the usual forced song pick, you’ll have to get together with a musician/musicians (or even be a producer yourself!) in teams of 2-8 (up to 5 mappers and up to 3 musicians) to create a fresh new song AND map!
+
+## Schedule
+
+| Phase | Dates |
+| --: | :-- |
+| Registration | Now to 2024-06-09 |
+| Song Production & Mapping Period | 2024-06-10 to 2024-08-22 |
+| Judging | 2024-08-23 to 2024-08-31 |
+| Results | 2024-09-1 |
+
+## Song Rules
+
+Starting June 10th, 2024, Musicians will be allowed to send their song to any of the organizers to check over for approval, under these rules:
+
+- **Songs must be original works created for the contest**, or at the very least, previously unreleased. Anything found to be breaking this rule will be immediately disqualified.
+- **Songs must stay unreleased until the day of results.**
+- **Songs must be 2:00-6:00 in drain time.**
+- **Obvious bait/troll submissions will be immediately disqualified.**
+- **No commissions allowed.** Creating an environment where people can out price others from having an artist is unjust.
+- **Teams just cannot be smaller than 2.** Mappers are allowed to be producers too!
+
+## Mapping Rules
+
+As soon as your team’s song is approved, you are allowed to start mapping!
+
+- **All submissions must include a single difficulty.**
+- **Drain time in the song should be split between all mappers participating as evenly as possible.**
+- **Submissions must be in osu! mode only.**
+- **All maps must follow the ranking criteria.**
+- **Storyboards/Videos are not permitted for the submission.** This is to prevent bias formed by the presentation of the map.
+
+## Submission
+
+Submissions will be handled through the [contest’s Mappers’ Guild Page](https://mappersguild.com/contests/listing?contest=6616268c100297322b373789), and will be anonymized upon submission, including artist names. A livestream may be held on the day of results to show the rankings!
+
+## Prizes
+
+| Ranking | Prizes |
+| :-- | :-- |
+| Winning Team | $100USD + Profile Badge |
```

Comment:
Potentially add contest points for the winning team, hyperlinked to https://osu.ppy.sh/wiki/en/Contests/Contest_points
```suggestion
| Winning Team | $100USD + Profile Badge + [Contest Points](/wiki/Contests/Contest_points) |
```